src/audio/heap.c

```diff
@@ -11,6 +11,8 @@
 
 #define ALIGN16(val) (((val) + 0xF) & ~0xF)
 
+extern u8 gIsVC;
```

Comment:
Could this line be in a header like game_init.h (or even vc_check.h if you move the declaration out of game_init.c)?

Reply:
moved to vc_check